QuickDeployment/QuickDeploymentClientSetup.md

```diff
@@ -2,41 +2,52 @@
 
 <!-- TOC -->
 
-- [Summary](#summary)
-- [DNS](#dns)
-- [Client Installation](#client-installation)
+- [Quick Deployment Client Setup](#quick-deployment-client-setup)
+    - [Summary](#summary)
+    - [DNS](#dns)
+    - [Client Installation](#client-installation)
 
 <!-- /TOC -->
 
 ## Summary
+
 Once you have QDE set up in your environment, you'll need to get clients talking to it. To do that, we'll need to do the following:
 
-* Setting up DNS on the client to access QDE.
-* Install the QDE SSL/TLS certificate so clients can access HTTPS components
-* Install Chocolatey and friends
+- Setting up DNS on the client to access QDE.
+- Install the QDE SSL/TLS certificate so clients can access HTTPS components
+- Install Chocolatey and friends
```

Comment:
Preference thing, but our markdown (minus the TOC) uses` *` - would prefer to continue that for consistency.

Reply:
So, we're keeping TOC with dashes, and everything else with `*`?

Reply:
yeah, that's what the thought was.

Reply:
Sounds good. That should be sorted in latest commit. 🙂